languages.js

```diff
@@ -12,6 +12,7 @@ var instructionsPtBr = require('./languages/translations/pt-BR.json');
 var instructionsRo = require('./languages/translations/ro.json');
 var instructionsRu = require('./languages/translations/ru.json');
 var instructionsSv = require('./languages/translations/sv.json');
+var instructionsSv = require('./languages/translations/tr.json');
```

Comment:
You overwrite Sweden language file with yours. You should take another various name, say `instructionsTr`